pkg/virtualKubelet/metrics/metrics.go

```diff
@@ -0,0 +1,64 @@
+package metrics
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// Metrics port
+	MetricsPort = ":9090"
+)
+
+var (
+	// ErrorsCounter is the counter of the errors occurred during the reflection.
+	ErrorsCounter *prometheus.CounterVec
+	// ItemsCounter is the counter of the items extracted from the workqueue and correctly processed.
+	ItemsCounter *prometheus.CounterVec
+	// MetricsLabels is the labels used for the metrics.
+	MetricsLabels []string
+)
+
+// Init initializes the metrics.
+func init() {
+	MetricsLabels = []string{"namespace", "reflector_type"}
```

Comment:
In my opinion "reflector_type" meaning is hard to understand, I suggest something like "reflected_resource" or similar